v2/tools/generator/internal/functions/testdata/TestGolden_PropertyAssignmentFunction_AsFunc/ConvertBetweenRequiredEnumAndRequiredEnum.golden

```diff
@@ -13,7 +16,7 @@ type Person struct {
 func (person *Person) AssignProperties_From_Person(source *vnext.Person) error {
 
 	// Release
-	release := Bucket(source.Release)
+	release := genruntime.ToEnum(string(source.Release), bucket_Values)
```

Comment:
where is `bucket_Values` actually defined?

Reply:
Internal to the package, next to the definition of the enum and its values. 

Here's an example from `v2/api/compute/v1api20220301
/virtual_machine_scale_set_types_gen.go`:

``` go
// +kubebuilder:validation:Enum={"Delete","Detach"}
type VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption string

const (
	VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption_Delete = VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption("Delete")
	VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption_Detach = VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption("Detach")
)

// Mapping from string to VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption
var virtualMachineScaleSetNetworkConfigurationProperties_DeleteOption_Values = map[string]VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption{
	"delete": VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption_Delete,
	"detach": VirtualMachineScaleSetNetworkConfigurationProperties_DeleteOption_Detach,
}
```